app/modules/login/LoginScreen.tsx

```diff
@@ -47,16 +48,19 @@ const LoginScreen = (): JSX.Element => {
             error={formik.errors.email}
           />
           <CustomInput
-            ref={ref => (formRefs.current[1] = ref)}
             onBlur={formik.handleBlur(Strings.formInputNames.password)}
             name={Strings.formInputNames.password}
+            ref={ref => (formRefs.current[1] = ref)}
```

Comment:
add optional chaining after current

Reply:
here I am not using current of formrefs, I am assigning ref of this input to formrefs.current[index]. so It is never going to cause any error in runtime. Because first we created empty array and then the ref of this input is getting stored at that particular index.